quick_start/kusto_sample_app.py

```diff
@@ -0,0 +1,334 @@
+# Todo - Start Here:

```

Comment:
why is it called Todo?

Reply:
There are specific comments where I want the user to intervene in the code by doing something.
Plus some IDEs highlight those differently.
Note the latest version has far few of those